client_configuration.go

```diff
@@ -100,18 +109,50 @@ func parseClientConfiguration(filePath string) (*ClientConfig, error) {
 	if err != nil {
 		return nil, parsingClientConfigError(err)
 	}
+	_, err = isCfgPermValid(filePath)
```

Comment:
Do we need first return value from this function if it is not used?

Reply:
We do not, removed it.